src/main/java/uk/gov/hmcts/darts/task/runner/impl/AbstractLockableAutomatedTask.java

```diff
@@ -79,7 +79,7 @@ private void setupUserAuthentication() {
     }
 
     @Override
-    public void run() {
+    public void run(boolean isManualRun) {
```

Comment:
This code is making my head hurt. Why are we passing this `isManualRun` in as a parameter? Can't we just use the `isManualTask` variable that is already used in this class?

Reply:
I have removed the ManualTaskService as this no longer provided any value and updated the code to reflect this change